server/text_generation_server/layers/moe/gptq_marlin.py

```diff
@@ -230,3 +232,111 @@ def _pack_weight(
     moe_weight.perm[expert] = weight.perm
 
     return moe_weight
+
+
+def fused_marlin_moe(
```

Comment:
I'd like to keep `moe` on the kernel hub as close to vLLM as possible, so moved this with our own extensions here.